yewprint-doc/src/app.rs

```diff
@@ -106,56 +106,68 @@ impl Component for App {
                                 />
                                 <MenuItem
                                 text={html!("Button")}
+                                href="#button"
                                 onclick=self.link.callback(|_| Msg::GoToMenu(DocMenu::Button))
                                 />
                                 <MenuItem
                                     text={html!("Button Group")}
+                                    href="#bgroup"
                                     onclick=self.link.callback(|_| Msg::GoToMenu(DocMenu::ButtonGroup))
```

Comment:
```suggestion
                                    href="#bgroup"
                                    onclick=self.link.callback(|_| Msg::GoToMenu(DocMenu::ButtonGroup))
```

:thinking: weird that it didn't break earlier